scripts/run-e2e.sh

```diff
@@ -2,23 +2,33 @@
 
 set -e
 
-export MESG_PATH="$(pwd)"/e2e.test/mesg
-
-# first run non existing test to detect compilation error quickly
+echo "run non existing test to detect compilation error quickly"
 go test -mod=readonly -v -count=1 ./e2e/... -run=__NONE__
 
 function onexit {
-  set +e
-  ./scripts/dev.sh stop
-  rm -rf "${MESG_PATH}"
+  docker service rm engine
+  docker wait $(docker ps -f label=com.docker.swarm.service.name=engine -q) 2> /dev/null
+
+  docker network remove engine
 }
 
 trap onexit EXIT
 
-rm -rf "${MESG_PATH}"
-mkdir -p "${MESG_PATH}"
-cp "$(pwd)"/e2e/testdata/e2e.config.yml "${MESG_PATH}"/config.yml
-
-./scripts/dev.sh -q
-
+if [[ -z $(docker network list -f name="engine" -q) ]]; then
+  docker network create --driver overlay engine
+fi
+docker service create \
+  --name engine \
+  -p 1317:1317 \
+  -p 50052:50052 \
+  -p 26657:26657 \
```

Comment:
is this port necessary? we shouldn't use it in e2e

Reply:
Tendermint RPC 26657 is mandatory to be able to run commands outside of the container and connect to it.
Eg: I'm developing something that use the tendermint RPC client.

The same comment apply to the orchestrator port 50052. It currently not necessary to publish it but it's mandatory to develop services on the host machine.